docs/fxa-engineering/fxa-sub-platform.md

```diff
@@ -169,13 +169,108 @@ server:
 
+[subscription endpoints]: https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#subscriptions
+
 ### Auth Server
 
 FxA's Auth Server makes Stripe API calls for authenticated FxA users via its [subscription
 endpoints][]. Stripe updates are sent back to the Auth Server via Stripe webhooks when a
 users subscription has been created/updated/deleted.
 
-[subscription endpoints]: https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#subscriptions
+Some Stripe webhooks will trigger emails.  These emails are behind a feature flag.  If you wish to send emails in your environment, set the auth server configuration
+
+```
+{
+  "subscriptions": {
+    "transactionalEmails": {
+      "enabled": true
+    }
+  }
+}
+```
+
+or the environment variable `SUBSCRIPTIONS_TRANSACTIONAL_EMAILS_ENABLED` to "true".  In order to receive Stripe webhook events in your local development, you need to use the [Stripe CLI](https://stripe.com/docs/stripe-cli/webhooks)'s event forwarding feature.
+
+## PayPal Integration
+
+PayPal can be configured as an additional payment provider in the Subscription Platform.  PayPal's [Express Checkout Reference Transactions](https://developer.paypal.com/docs/archive/express-checkout/integration-guide/ECReferenceTxns/) is the feature that enables the Subscription Platform to use PayPal as a payment provider for recurring subscriptions.  The customer's PayPal Billing Agreement ID is saved for future subscription invoices.  See [the diagram below](#paypal-checkout) for details on this process.
+
+The PayPal paid subscriptions are still driven by Stripe's subscription and invoicing model.  The key difference is that PayPal paid subscriptions have a [collection method](https://stripe.com/docs/api/subscriptions/object#subscription_object-collection_method) of `send_invoice`.  The [PayPal processor](#paypal-processor) is used to pay the inovices with the customer's billing agreement ID.
+
+### PayPal Accounts
+
+For development, you will need a PayPal developer account.  Once you are in the PayPal developer dashboard, navigate to "Accounts" under the Sandbox section of the menu.  Here you should create a pair of personal and business sandbox accounts.  The personal account will be use to test as the customer; the business account will be receiving the subscription payments.
+
+Once you've added the account pair, navigate to the bussines account by selecting "View/edit account".  Now click on the "API Credentials" tab.  You'll need the "NVP/SOAP Sandbox API Credentials" for the next section.
+
+### Configuration
+
+#### Auth Server
+
+In order to enable and use PayPal in the auth server, set the following configuration options
+
+```
+{
+  "subscriptions": {
+    "paypalNvpSigCredentials": true,
+    "sandbox": true,
+    "user": "your PayPal NVP API User name",
+    "pwd": "your PayPal NVP API password",
+    "signature": "your PayPal NVP API signature"
```

Comment:
These are unfortunately pretty hard to find in the PayPal dashboard. I actually gave some instructions on how to find them in the [fxa-auth-server README](https://github.com/mozilla/fxa/tree/main/packages/fxa-auth-server#secrets) -- maybe this is a better place for that?:
> The sandbox PayPal business account API credentials above can be found in the PayPal developer dashboard under "Sandbox" > "Accounts". You may need to create a business account if one doesn't exist.

Unfortunately I haven't been able to find anything in PayPal's docs that talks specifically about this -- only [this page](https://developer.paypal.com/docs/nvp-soap-api/apiCredentials/#api-signatures) which refers to requesting an API signature.

Reply:
I also mentioned it above in the paragraph starting "Once you've added the account pair, navigate to the bussines account..."  Hmmm this would be an argument against moving the PayPal config section, since that paragraph about PayPal accounts leads into it.